.github/workflows/verify-push.yaml

```diff
@@ -19,7 +19,22 @@ jobs:
   build-and-push:
     name: Build and push image to Quay.io
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        install_latest: [ true, false ]
+
     steps:
+      - name: Install latest podman
+        if: matrix.install_latest
+        # https://podman.io/getting-started/installation
+        run: |
+          . /etc/os-release
```

Comment:
Instead of duplicating these steps across all our workflows, we can commit a script that captures them to the root of the repository or to `.github/`